scripts/exglobal_land_analysis_run.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
I think the g-w preference is to use python for UFS-DA ex-scripts.

Reply:
@RussTreadon-NOAA There are many processes/runs in this run script `scripts/exglobal_land_analysis_run.sh` for the snow DA, such as: 1) create the ensembles; 2) process the IMS data; 3) conduct JEDI runs; and 4) apply JEDI increments. We have discussed and agreed to use the shell scripts here for this snow DA implementation initially due to the limited time.

Reply:
If the g-w team approves of this approach, I'm fine with their decision.